### ISSUE TITLE
Pagination: Moves position

### ISSUE THREAD

**Issue by Maximilian-R** [seb-oss/green#790]
Tested it out and noticed some issues:

### Issue 1
When on page 1 and clicking on next arrow, the prev-arrow is added and push the buttons so if I click again w/o moving mouse I go to last page.

<img width="349" alt="Screenshot 2023-03-02 at 12 39 04" src="https://user-images.githubusercontent.com/11420341/222419108-4fbaa25b-4b7b-418b-8f41-02d31207132d.png">

<img width="383" alt="Screenshot 2023-03-02 at 12 39 08" src="https://user-images.githubusercontent.com/11420341/222419009-43955f45-9be5-4293-8eeb-7e0935d1b116.png">

Maybe arrows should always keep it's space? 

The dots are also changing right arrow position, but maybe not a problem.

### Issue 2

Doesn't seem to work with keyboard navigation (Enter or Space) to select pages or prev/next

_Originally posted by @vsjolander in https://github.com/sebgroup/green/pull/735#pullrequestreview-1321664359_

**Comment by EldRoos** [comment]
I agree it is troublesome it changes position and width depending on what is selected.

Looks like enter now works (but not space).

**Comment by EldRoos** [comment]
A new core component will be made. Closing this.
